src/components/forms/OfficeHoursForm.tsx

```diff
@@ -277,6 +301,13 @@ export const OfficeHoursForm: FC = () => {
                     </PageText>
                   </Box>
                 )}
+                {errors?.company?.type === 'validate' && (
+                  <Box mt={1}>
+                    <PageText as='small' fontSize='helpText' color='red.500'>
+                      Organization name contain a URL.
```

Comment:
For consistency within the error messages and more clarity, we should rephrase it as 'Organization name cannot contain a URL'